[docs] Add Results on the top of the TPC-C page. (#4949)
yugabyte/yugabyte-db: docs/content/latest/benchmark/tpcc-ysql/_index.md

## Overview
Follow the steps below to run the open-source [oltpbench](https://github.com/oltpbenchmark/oltpbench) TPC-C workload against YugabyteDB YSQL. [TPC-C](http://www.tpc.org/tpcc/) is a popular online transaction processing benchmark that provides metrics you can use to evaluate the performance of YugabyteDB for concurrent transactions of different types and complexity that are either either executed online or queued for deferred execution.

### Results at a glance

> | Warehouses| TPMC | Efficiency

Reviewer: * Can we rename `Efficiency` to `Efficiency (approx)`?

* We should also add the following column:

**Cluster Size**
The values for this should look like the following:
3 nodes of type `c5d.4xlarge` (16 vCPUs)

* Also, can we change the 10 warehouses example to a 2 core machine? Say a c5.large or something like that. Since its just an example (assuming our efficiency will be close to max here).

Author: Done.

-------------|-----------|------------|------------|

> 10   | 126   | 97.97% | 3 nodes of type `c5.large` (2 vCPUs)

Reviewer 2: Any reason why we didn't use c5d.large here instead of c5.large?

Author: We wanted to use a 2vCPU machine to indicate that the workload can run on small machines.

Reviewer 2: c5d.large is also 2 vCPU, right? We decided to move away from c5.xx and use c5d.xx.

{{< /tip>}}

## 2. Configure DB connection params (optional)

> The details like IPs of the nodes, number of warehouses and number of loader threads are controlled by command line arguments.

Reviewer 2: ```suggestion
Workload configuration like IP addresses of the nodes, number of warehouses and number of loader threads can be controlled by command line arguments.
```

Author: Done

> For other options like username, password, port, etc., you can change them using the configuration file at `config/workload_all.xml`, if not working with the defaults.

Reviewer 2: ```suggestion
Other options like username, password, port, etc. can be changed them using the configuration file at `config/workload_all.xml`, if needed.
```

Author: Done.

{{< /tip>}}

## 2. Configure DB connection params (optional)

If not working with the defaults, we can change the username, password, port, etc. using the configuration file at `config/workload_all.xml`. We can also change the terminals or the physical connections being used by the benchmark using the configuration.

> 

Reviewer 2: Let's remove the options that don't need to be changed by users - specifically, `dbtype`, `driver`, `terminals` and `numDBConnections`.

terminals and numDBConnections is something that we should set automatically based on number of warehouses. It's suboptimal and unnecessary for users to change this.

Please also add command line options like loader threads and warehouses to this list and change the title to `Configure workload parameters`

> By default the number of terminals is 10 times the number of warehouses which is the max that the TPC-C spec allows. The number of DB connections is the same as the number of warehouses.

Author: We can change it using the config.
Although we ideally want them to change it.
